lib/commands/touch.js

```diff
@@ -1,9 +1,51 @@
 const commands = {};
-
 //This is needed to make clicks on -image elements work properly
 commands.performActions = async function (actions) {
-  return await this.winAppDriver.sendCommand('/actions', 'POST', {actions});
+  switch (actions[0].id) {
+    case 'default keyboard':
```

Comment:
winappdriver does support w3c actions except of these targeted for mouse inout device (touch and pen are supported). Su I would rather distinguish based on the input device type

Reply:
that's what the id field contains, the input device identifier. Default keyboard and default mouse input types are not supported

Reply:
id field may contain any non-empty string. The standard does not limit that. The list of available input sources is limited though: https://www.w3.org/TR/webdriver/#input-sources

Reply:
you're right, will modify the check